proposals/a-template/respec-support/specification-wrapper.html

```diff
@@ -13,29 +13,37 @@
     <script src='contributors.js' class='remove'></script>
     <script class='remove'>
       var respecConfig = {
+        mainSpecTitle: "Open Badges",
+        mainSpecBiblioKey: "OB-20",
         specTitle: "ProperProjectName",
         shortName: "ShortProjectName",
         specNature: "normative", // spec nature is "normative" or "informative"
-        specType: "main", // spec type is "main", "cert", "impl", "errata" or other agreed upon string
+        specType: "proposal", // spec type is "main", "cert", "impl", "errata" or other agreed upon string
         specVersion: "0.1",
-        specStatus: "Draft Proposal", // specStatus: IMS Base Document, IMS Candidate Final (Public) or IMS Final Release
+        specStatus: "IMS Base Document", // specStatus: IMS Base Document, IMS Candidate Final (Public) or IMS Final Release
         specDate: "Never 12th, 2240",
-        format: "markdown",
         contributors: _contributors,
-        localBiblio: _localBiblio,
+        localBiblio: _localBiblio
       };
     </script>
   </head>
   <body>
-    <section class="introduction" id="abstract" data-format="markdown">
-      <h2>Abstract</h2>
```

Comment:
Could put the <h2>Abstract</h2> here or ## Abstract in the markdown. I prefer putting it in the markdown to match the other markdown files.